linera-base/src/data_types.rs

```diff
@@ -780,6 +780,8 @@ pub enum OracleResponse {
     Assert,
     /// The block's validation round.
     Round(Option<u32>),
+    /// An event was read.
+    Event(EventId, Vec<u8>),
```

Comment:
I thought that `OracleResponses` are for things that are non-deterministic and we need to either enforce or check that they match. Why is reading an event - by its id - an oracle event?

Reply:
Because a validator may or may not have seen that event yet.

In the future we will probably write client code to update validators about missing events, but for now it is sufficient if everyone synchronizes the admin chain.

Reply:
events are results of executing a block - won't validator _have to_ see the event if he executed the block (has the certificates)?

Reply:
And you're planning to use it for what exactly? User chains checking for committee/epoch changes on the admin chain?

I understand the desire to make the follow-up PRs easier to review but it's hard to understand/review these changes in isolation b/c we're basically reviewing part of a design w/o a design doc and full picture.

Reply:
OK, let me undo this part. Maybe this belongs in the follow-up PR.

(A user chain processing an epoch change will create this oracle response, yes.)

Reply:
I removed it, and updated the PR description as discussed.